core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +157,16 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.getExplorationInteractions = function() {
+          $http.post(GET_INTERACTIONS_BY_ID_HANDLER_URL)
+            .then(function(response) {
+              ctrl.setStatusMessage('Success! ' + response.data.title);
+            }, function(errorResponse) {
+              ctrl.setStatusMessage(
+                'Server error: ' + errorResponse.data.error);
```

Comment:
@seanlip getting error as missing id/name can you help with this part

Reply:
You need to debug things like this. Try and trace what code is running, and provide your analysis here.

Reply:
Okay i will try but other than this is the correct way of doing the intended work ? Or i an doing something wrong

Reply:
Also i tried to run with a hardcoded id in that case it worked just fine so probably something wrong in admin.py i guess

Reply:
The rough architecture seems reasonable (you're following existing patterns in the code) but it's hard to give you more useful feedback until you flesh out more of the details.